cmd/common/cmdline.go

```diff
@@ -33,8 +33,14 @@ type GlobalFlags struct {
 }
 
 func NewGlobalFlags() *GlobalFlags {
+	var name string
+	if os.Args == nil {
```

Comment:
Is it *actually* possible for `os.Args` to be `nil`? It can technically be `nil` because it's a byte slice, but the docs are fairly definitive: `Args hold the command-line arguments, starting with the program name.` This doesn't hurt anything, though.